src/logic/calculate.test.js

```diff
@@ -0,0 +1,5 @@
+import calculate from './calculate';
+
+it('Calculte should work without error', () => {
+  expect(calculate.length).toBe(2);
+});
```

Comment:
- [ ] All the functionalities & scenarios should be tested, e.g, test when `buttonName` is `AC`, `0`, `.`, `=`, `+/-`, and so on. Also test for when `obj` and its attributes like `obj.next`, `obj.total`, and `obj.operation` are present or have different values.

Reply:
Thank you @cyonii

Reply:
Well done